data/_models/shared/Processes.csv

```diff
@@ -1,119 +1,143 @@
+Measurements,,variable,string,1,TRUE,,,,,,,http://purl.obolibrary.org/obo/NCIT_C54166,"A name that describes the measurement (e.g.. ""Blood platelet count"")","Patient registry, FAIR Genomes"
+Measurements,,value,int,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C25712,"The value that was measured, collected, derived, etc. (e.g. ""55345"")","Patient registry, FAIR Genomes"
+Measurements,,unit,ontology,,,CatalogueOntologies,Units,,,,,http://purl.obolibrary.org/obo/NCIT_C25709,"A value that specifes the measured value (e.g., ""Per Millimeter"")","Patient registry, FAIR Genomes"
+Metabolomic material processing,Processes,,,,,,,,,,,http://purl.obolibrary.org/obo/OBI_0000073,A metabolomics material processing is a protocol application including material enrollments and biomaterial transformations.,FAIR Genomes metabolomics add-on
+Metabolomic material processing,,processing type,ontology,,,CatalogueOntologies,Metabolomic processing type,,,,,http://purl.obolibrary.org/obo/OBI_0000094,A planned process which results in physical changes in a specified input material.,FAIR Genomes metabolomics add-on
+Metabolomic material processing,,sample extraction,string,,,,,,,,,http://purl.obolibrary.org/obo/CHMO_0001577,The transfer of a solute from a liquid phase to another immiscible or partially-miscible liquid phase in contact with it.,FAIR Genomes metabolomics add-on
+Metabolomic material processing,,filter used,string,,,,,,,,,http://purl.obolibrary.org/obo/CHMO_0001640,"The filter used in separating of suspended solids from a liquid or gas, usually by forcing a carrier gas or liquid through a porous medium.",FAIR Genomes metabolomics add-on
+Metabolomic material processing,,passing rate,int,,,,,,,,,http://purl.obolibrary.org/obo/CHMO_0002751,The volume of carrier gas passing through a chromatography column in unit time.,FAIR Genomes metabolomics add-on
+Metabolomic material processing,,SPE cartridge,string,,,,,,,,,http://purl.obolibrary.org/obo/OMIT_0025161,"A sample preparation technique using a solid adsorbent contained most commonly in a cartridge device, or on a disk to adsorb select species from solution.",FAIR Genomes metabolomics add-on
+Metabolomic material processing,,washing protocol,text,,,,,,,,,http://purl.obolibrary.org/obo/OBI_0302888,Washing is a process by which a material entity acting as contaminant (e.g. excess staining reagent) is removed by application of one or more cycles of solution in flow.,FAIR Genomes metabolomics add-on
+Metabolomic mass spec assays,Processes,,,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C60819,A qualitative or quantitative analysis performed to determine the amount of a particular constituent in a sample or the biological or pharmacological properties of a drug.,FAIR Genomes metabolomics add-on
```

Comment:
Then here again question like in DNA seq if we want to merge Metabolic material processing with the mass spec assay. Largely depends on if there is a many-to-one (many mass spec on one material) or if it is more like one-to-one (as in sequencing mostly).